modules/ROOT/pages/choice-router-concept.adoc

```diff
@@ -85,6 +87,50 @@ http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/htt
 </mule>
 ----
 
+== Adding Additional Route Options
```

Comment:
```suggestion
== Adding Route Options
```

Reply:
Actually, now that I see the subtopic, I suggest two Level-2 topics:
==Add Routes by Dragging Components in Studio
==Add Routes by Editing the XML
Maybe they should be Level-3 topics within a Level-2 topic named "Adding Routes to the Choice Router," which would have the text explaining the typical case of two routes and then why you might want to add others?